process/block/preprocess/transactions.go

```diff
@@ -1372,7 +1372,7 @@ func (txs *transactions) sortTransactionsBySenderAndNonceWithFrontRunningProtect
 	xoredAddresses := make(map[string][]byte)
 
 	for _, tx := range transactions {
-		xoredAddresses[string(tx.Tx.GetSndAddr())] = xorBytes(tx.Tx.GetSndAddr(), randSeed)
+		xoredAddresses[string(tx.Tx.GetSndAddr())] = txs.hasher.Compute(string(xorBytes(tx.Tx.GetSndAddr(), randSeed)))
```

Comment:
could have been broken in 2 instructions:
```GO
xorBytes := xorBytes(tx.Tx.GetSndAddr(), randSeed)
xoredAddresses[string(tx.Tx.GetSndAddr())] = txs.hasher.Compute(string(xorBytes))
```

Reply:
done